include/deal.II/lac/sparse_decomposition.h

```diff
@@ -237,24 +237,6 @@ class SparseLUDecomposition : protected SparseMatrix<number>,
                    const AdditionalData parameters);
 
   /**
-   * This method is deprecated, and left for backward compability. It will be
-   * removed in later versions.
-   *
-   * @deprecated
-   */
-  template <typename somenumber>
-  void decompose (const SparseMatrix<somenumber> &matrix,
-                  const double                    strengthen_diagonal=0.) DEAL_II_DEPRECATED;
-
-  /**
-   * This method is deprecated, and left for backward compability. It will be
-   * removed in later versions.
-   *
-   * @deprecated
-   */
-  virtual bool is_decomposed () const DEAL_II_DEPRECATED;
-
-  /**
```

Comment:
The documentation of the 'decomposed' flag (protected data member, line 313) still refers to decompose(). Could you please remove that reference and simply state that the derived classes will use this flag for tracking the state of the decomposition?